src/app/src/components/DashboardAdjustMatchCard.jsx

```diff
@@ -369,6 +379,21 @@ export default function DashboardAdjustMatchCard({
                                 label="Country Code"
                                 value={match.country_code}
                             />
+                            <MatchDetailItem
+                                label="Status"
+                                value={
+                                    match.status[0] +
+                                    match.status.slice(1).toLowerCase()
```

Comment:
Consider `import capitalize from 'lodash/capitalize'` so it it easier to see at a glance how the status is being formatted.

Reply:
Updated